plonetheme/onegovbear/theme/scss/form.scss

```diff
@@ -25,6 +25,7 @@ $input-error-border-color: transparentize($danger-color, .5) !default;
   padding: 0.5em;
   width: 20em;
   outline: none;
+  box-sizing: border-box;
```

Comment:
@jone Due to https://github.com/twbs/bootstrap/blob/3487898a6af8c1d5b12f7b040d30e71b19e34923/less/scaffolding.less#L12 and https://css-tricks.com/box-sizing/ ```box-sizing: border-box``` is the thing you always want. So let apply this to all input types and look for destructive things and fix them. ftw.theming is based on bootstrap anyway.